src/Resources/config/services.xml

```diff
@@ -2,66 +2,25 @@
         </parameter>
     </parameters>
 
     <services>
-        <service id="setono.sylius_redirect.form.type.redirect"
-                 class="Setono\SyliusRedirectPlugin\Form\Type\RedirectType">
-            <argument>%setono_sylius_redirect.model.redirect.class%</argument>
-            <argument>%setono_sylius_redirect.form.type.redirect.validation_groups%</argument>
-            <tag name="form.type"/>
-        </service>
-
-        <service id="Setono\SyliusRedirectPlugin\Command\RemoveRedirectsCommand">
-            <argument type="service" id="setono_sylius_redirect.repository.redirect"/>
-            <argument>%setono_sylius_redirect.remove_after%</argument>
-            <tag name="console.command"/>
-        </service>
-
-        <!-- Event Listeners -->
-        <service id="Setono\SyliusRedirectPlugin\Menu\AdminMenuListener">
-            <tag name="kernel.event_listener" event="sylius.menu.admin.main" method="addAdminMenuItems"/>
-        </service>
-
-        <service id="Setono\SyliusRedirectPlugin\EventListener\NotFoundSubscriber">
-            <argument type="service" id="setono_sylius_redirect.manager.redirect"/>
-            <argument type="service" id="sylius.context.channel"/>
-            <argument type="service" id="setono_sylius_redirect.resolver.redirection_path"/>
-            <tag name="kernel.event_subscriber"/>
-        </service>
-
-        <service id="Setono\SyliusRedirectPlugin\EventListener\ControllerSubscriber">
-            <argument type="service" id="setono_sylius_redirect.manager.redirect"/>
-            <argument type="service" id="sylius.context.channel"/>
-            <argument type="service" id="setono_sylius_redirect.resolver.redirection_path"/>
-            <tag name="kernel.event_subscriber"/>
-        </service>
-
-        <!-- Constraints -->
-        <service id="Setono\SyliusRedirectPlugin\Validator\Constraints\InfiniteLoopValidator">
-            <argument type="service" id="sylius.repository.channel"/>
-            <argument type="service" id="setono_sylius_redirect.resolver.redirection_path"/>
-            <tag name="validator.constraint_validator"/>
-        </service>
-
-        <service id="Setono\SyliusRedirectPlugin\Validator\Constraints\SourceRegexValidator">
-            <argument>%sylius.security.shop_regex%</argument>
-            <tag name="validator.constraint_validator"/>
-        </service>
-
-        <service id="Setono\SyliusRedirectPlugin\Validator\Constraints\SourceValidator">
-            <argument type="service" id="setono_sylius_redirect.repository.redirect"/>
-            <tag name="validator.constraint_validator"/>
-        </service>
-
-        <!-- Resolvers -->
-        <service id="setono_sylius_redirect.resolver.redirection_path"
-                 class="Setono\SyliusRedirectPlugin\Resolver\RedirectionPathResolver">
-            <argument type="service" id="setono_sylius_redirect.repository.redirect"/>
-        </service>
+        <defaults public="false"/>
```

Comment:
It false by default, AFAIK

Reply:
Indeed. It was introduced in Symfony 3.4. I can remove this useless line.